n_helpers.c

```diff
@@ -441,14 +423,51 @@ const char * NoteBinaryReceive(uint8_t * buffer, uint32_t bufLen,
     char hashString[NOTE_MD5_HASH_STRING_SIZE] = {0};
     NoteMD5HashString(buffer, decLen, hashString, NOTE_MD5_HASH_STRING_SIZE);
     if (strncmp(hashString, status, NOTE_MD5_HASH_STRING_SIZE)) {
-        NOTE_C_LOG_ERROR("computed MD5 does not match received MD5");
-        return ERRSTR("computed MD5 does not match received MD5", c_err);
+        const char *err = ERRSTR("computed MD5 does not match received MD5", c_err);
+        NOTE_C_LOG_ERROR(err);
+        return err;
     }
 
     // Return `NULL` if success, else error string pointer
     return NULL;
 }
 
+//**************************************************************************/
+/*!
+  @brief  Reset the Notecard's binary buffer.
```

Comment:
Should we call it the binary data store / binary store instead of binary buffer? Same question applies elsewhere where we use the term binary buffer.